src/main/java/ee/tuleva/onboarding/epis/mandate/MandateDto.java

```diff
@@ -39,6 +40,8 @@ public class MandateDto {
 
   private String phoneNumber;
 
+  private Optional<BigDecimal> paymentRate;
```

Comment:
We should separate the paymentRate domain from the Mandate domain as much as possible.
See how the cancellation mandates are separete: ApplicationController.cancel()

Reply:
I propse we create a new endpoint like `/v1/2nd-pillar-payment-rates` where you can POST a `{ "paymentRate": 6 }` and get a mandateId in response. See ApplicationController.cancel() for an example.

Reply:
Not sure how to make it RESTful though...